### PR TITLE
Implement some basis types needed for other types and data structures.

### PR DESCRIPTION
**Pull request by skhugh**
- Implements CrdtElement, ActorID and TimeTicket classes which are needed for other types and data structures such as RGATreeList and RHT.
- Changes names for acronyms to conform to following Kotlin conventions:

> When using an acronym as part of a declaration name, capitalize it if it consists of two letters (IOStream); capitalize only the first letter if it is longer (XmlFormatter, HttpInputStream).

#### What are the relevant tickets?

Fixes #

### Checklist

- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

**Review by hackerwins — APPROVED** [review]
LGTM. 👍

File: yorkie/src/main/kotlin/dev/yorkie/document/time/ActorID.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package dev.yorkie.document.time
+
+/**
+ * [ActorID] is used to identify who is making changes to the document.
+ * It is hexadecimal string and should be generated by unique value.
+ */
+@JvmInline
+internal value class ActorID(val id: String) : Comparable<ActorID> {
```
Comment (hackerwins) [inline yorkie/src/main/kotlin/dev/yorkie/document/time/ActorID.kt:8]: Initials are expressed as Uppercase in Go. Do we have to express `ActorID` as `ActorId` in Kotlin?

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Comment (skhugh) [reply yorkie/src/main/kotlin/dev/yorkie/document/time/ActorID.kt:8]: [Kotlin doc states](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names) that we should use all uppercase for 2-letter acronyms and camel case for more letters.
So it seems `ActorID` is the better choice.

Comment (daeyounglnc) [reply yorkie/src/main/kotlin/dev/yorkie/document/time/ActorID.kt:8]: According to [Kotlin doc states](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names), it seems that both `CrdtArray` and `CrdtElement` classes should be changed to `CRDT~`, isn't it?

Comment (hackerwins) [reply yorkie/src/main/kotlin/dev/yorkie/document/time/ActorID.kt:8]: I saw the examples from the link left by @skhugh.

> When using an acronym as part of a declaration name, capitalize it if it consists of two letters (IOStream); capitalize only the first letter if it is longer (XmlFormatter, HttpInputStream).

**Pull request merged**